conseil-common/src/main/scala/tech/cryptonomic/conseil/common/ethereum/rpc/EthereumRpcClient.scala

```diff
@@ -72,7 +77,7 @@ class EthereumClient[F[_]: Concurrent](
       stream
         .map(_.hash)
         .map(EthGetTransactionReceipt.request)
-        .through(client.stream[EthGetTransactionReceipt.Params, TransactionReceipt](batchSize = 1))
+        .through(client.stream[EthGetTransactionReceipt.Params, TransactionReceipt](batchSize = 10))
```

Comment:
what is the reason for `batchSize` change? If it's performance I think we could extract it into the configuration.